test/customers/get-customers.api-test.ts

```diff
@@ -148,10 +148,6 @@ describe('GET /customers', () => {
       query: { name: valueGenerator.string({ length: 256 }) },
       expectedError: 'name must be shorter than or equal to 255 characters',
     },
-    {
-      query: { name: valueGenerator.word(), extraParameter: valueGenerator.word() },
-      expectedError: 'property extraParameter should not exist',
-    },
```

Comment:
Why has this been removed?

Reply:
Test was expecting error on for unexpected extra parameters.

I changed global validation settings to match other APIM NodeJS applications.

![image](https://github.com/UK-Export-Finance/mdm-api/assets/2437608/d10de112-1d11-4f7c-a7aa-a2680dc1ac1d)

Now extra parameters doesn't throw errors and test is not valid anymore.